js/downloads.js

```diff
@@ -1,31 +1,26 @@
-// Script for generating Spark download links
-// No dependencies; pure javascript.
-
-releases = {};
+const releases = {};
 
 function addRelease(version, releaseDate, packages, mirrored) {
   releases[version] = {
     released: releaseDate,
-    packages: packages,
-    mirrored: mirrored
+    packages,
+    mirrored
   };
 }
 
-var sources = {pretty: "Source Code", tag: "sources"};
-var hadoopFree = {pretty: "Pre-built with user-provided Apache Hadoop", tag: "without-hadoop"};
-var hadoop2p7 = {pretty: "Pre-built for Apache Hadoop 2.7", tag: "hadoop2.7"};
-var hadoop3p3 = {pretty: "Pre-built for Apache Hadoop 3.2 and later", tag: "hadoop3.2"};
-var hadoop3p3scala213 = {pretty: "Pre-built for Apache Hadoop 3.2 and later (Scala 2.13)", tag: "hadoop3.2-scala2.13"};
-var hadoop2p = {pretty: "Pre-built for Apache Hadoop 2.7", tag: "hadoop2"};
-var hadoop3p = {pretty: "Pre-built for Apache Hadoop 3.3 and later", tag: "hadoop3"};
-var hadoop3pscala213 = {pretty: "Pre-built for Apache Hadoop 3.3 and later (Scala 2.13)", tag: "hadoop3-scala2.13"};
-
-// 3.2.0+
-var packagesV12 = [hadoop3p3, hadoop3p3scala213, hadoop2p7, hadoopFree, sources];
-// 3.3.0+
-var packagesV13 = [hadoop3p, hadoop3pscala213, hadoop2p, hadoopFree, sources];
-
-addRelease("3.3.2", new Date("02/17/2023"), packagesV13, true);
+const sources = {pretty: "Source Code", tag: "sources"};
+const hadoopFree = {pretty: "Pre-built with user-provided Apache Hadoop", tag: "without-hadoop"};
+const hadoop2p7 = {pretty: "Pre-built for Apache Hadoop 2.7", tag: "hadoop2.7"};
+const hadoop3p3 = {pretty: "Pre-built for Apache Hadoop 3.2 and later", tag: "hadoop3.2"};
+const hadoop3p3scala213 = {pretty: "Pre-built for Apache Hadoop 3.2 and later (Scala 2.13)", tag: "hadoop3.2-scala2.13"};
+const hadoop2p = {pretty: "Pre-built for Apache Hadoop 2.7", tag: "hadoop2"};
+const hadoop3p = {pretty: "Pre-built for Apache Hadoop 3.3 and later", tag: "hadoop3"};
+const hadoop3pscala213 = {pretty: "Pre-built for Apache Hadoop 3.3 and later (Scala 2.13)", tag: "hadoop3-scala2.13"};
+
+const packagesV12 = [hadoop3p3, hadoop3p3scala213, hadoop2p7, hadoopFree, sources];
+const packagesV13 = [hadoop3p, hadoop3pscala213, hadoop2p, hadoopFree, sources];
+
+addRelease("3.3.3", new Date("04/14/2023"), packagesV13, true);
```

Comment:
I am wondering why `3.3.3` is associated with `04/14/2023`. The date should be the release date.

Reply:
Ya, Apache Spark community doesn't release it yet.